README.adoc

```diff
@@ -43,6 +43,21 @@ This mode of operation will be used in production.
 
 To build and run the non-simulated version of Omicron, see: xref:docs/how-to-run.adoc[].
 
+== Run tests with nextest
+
+The supported way to run tests is via https://nexte.st/[cargo-nextest].
```

Comment:
I like the idea of heading off the inevitable cargo test question by saying something like 

> `cargo test` may work, but we can't guarantee that as we do not run it in CI.

Reply:
Done, I think that makes sense.